application/src/main/java/org/togetherjava/tjbot/features/reminder/RemindRoutine.java

```diff
@@ -147,34 +134,29 @@ static ReminderRoute toPrivate(PrivateChannel channel) {
         }
     }
 
-    private void reminderFailureAction(int id, long channelId, long authorId,
-                                       CharSequence content, long guildId, int reminderFailureCounter) {
-        if (reminderFailureCounter <= 3) {
-            Instant current = Instant.now();
-            database.write(context -> context.newRecord(PENDING_REMINDERS)
-                    .setId(id)
-                    .setCreatedAt(Instant.now())
-                    .setGuildId(guildId)
-                    .setChannelId(channelId)
-                    .setAuthorId(authorId)
-                    .setRemindAt(current.plusSeconds(60))
-                    .setContent(content.toString())
-                    .setReminderFailureCounter(reminderFailureCounter + 1)
-                    .insert());
-        } else {
+    private void attemptRetryReminder(Reminder reminder) {
+        if (!(reminder.failureAttempts() <= MAX_FAILURE_RETRY)) {
```

Comment:
kinda complex to read. instead do `foo > bar`

Reply:
yep just fixed this one, idk why i went with that without thinking why you suggested what you suggested earlier. Over complicated it for no reasons.